docs/publish.js

```diff
@@ -80,6 +80,7 @@ function generate(title, content, filename, template) {
     const outpath = path.join(outDir, filename);
     let html = view.render(template || 'page.tmpl', { env, title, content });
     html = helper.resolveLinks(html);
+    html = html.replace(/`([\S ]*)`/g, '<code>$1</code>');
```

Comment:
This should use [_non-greedy_ matching](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Quantifiers) (i.e. `*?` rather than `*`), otherwise something like
```
Some `text` with `two` code snippets
```
will be replaced by
```
Some <code>text` with `two</code> code snippets
```

Reply:
Ah yes, in fact I used `/<\/code>/` when replacing them by a ` ! Changing that

Reply:
Yep, thanks for noticing this